src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Diagnostics.RuntimeHost.Services.DevOpsClient;
+using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Diagnostics.RuntimeHost.Controllers
+{
+    [Authorize]
+    [Produces("application/json")]
+    [Route(UriElements.DevOps)]
+    public class DevOpsController : Controller
+    {
+        IDevOpsClient _devOpsClient;
+
+        public DevOpsController(IConfiguration configuration, IDevOpsClient devOpsClient)
+        {
+            _devOpsClient = devOpsClient;
+        }
+
+        [HttpGet(UriElements.DevOpsMakePR)]
+        public async Task<IActionResult> makePullRequestAsync(string sourceBranch, string targetBranch, string title)
+        {
+            DevOpsResponse response = await _devOpsClient.makePullRequestAsync(sourceBranch, targetBranch, title);
```

Comment:
How does AppLens client know which target branch to make PR against?

Reply:
It gets it from user input